packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -214,7 +230,10 @@ function unpack(message: ISequencedDocumentMessage): InboundSequencedContainerRu
  *
  * @internal
  */
-export function unpackRuntimeMessage(message: ISequencedDocumentMessage): boolean {
+export function unpackRuntimeMessage(
+	message: ISequencedDocumentMessage,
+	logLegacyCase: (codePath: string) => void = () => {},
+): boolean {
 	if (message.type !== MessageType.Operation) {
```

Comment:
@kian-thompson -- If the _original_ message didn't have type "op", is it _for sure_ that it's not chunked, compressed or grouped?

If there's any doubt there, then that "non-modern" message could be transformed by `RemoteMessageProcessor.process` and then _not_ fall into this early return.  In that case, the whole previous refactor to avoid calling `RemoteMessageProcessor` for "non-modern" is flawed.

Can you confirm? I can dig through source history, but if you know confidently or can quickly point back at history to prove it that would be great.

Reply:
I can't think of a case where an op is chunked, compressed, or grouped but wouldn't have wrapped type "op". And those type of version compat differences wouldn't be supported (new runtime, really old loader).

The only _maybe_ would be legacy chunking, which our e2e test uses version "0.56.0" for the runtime.